tools/files.js

```diff
@@ -643,7 +642,7 @@ files.freeTempDir = function (tempDir) {
       // Don't crash and print a stack trace because we failed to delete a temp
       // directory. This happens sometimes on Windows and seems to be
       // unavoidable.
-      Console.debug(err);
```

Comment:
I was pairing with Slava yesterday and we got a Console is undefined error that we didn't understand. The calls happen in a weird context (responding to a signal and clearing up), so that may have something to do with it (although I have no idea what that would do).